### PR TITLE
Plugins: Fix plugin query help markdown 

### PR DESCRIPTION
**Pull request by wbrowne**
**What is this feature?**

**Why do we need this feature?**

A bug was introduced alongside https://github.com/grafana/grafana/pull/58264, as `readme` was removed as the markdown fallback, and it was replaced with `help`. This meant that the behaviour of `<PluginHelp>` component was changed for all plugins that does not include a `query_help.md` - instead of being able to fallback to `readme.md`, it tried `test.md` which does not appear to be a convention in Grafana (any longer? ever?).

Where the component is used:
Screenshot 2022-12-16 at 17 34 38

A successful fallback to README:
image

**Who is this feature for?**

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by marefr — APPROVED** [review]
LGTM

File: public/app/core/components/PluginHelp/PluginHelp.tsx [hunk 1]
```diff
@@ -1,83 +1,28 @@
...
-    if (isError) {
-      return <h3>&apos;Error occurred when loading help&apos;</h3>;
-    }
-
-    if (type === 'panel_help' && help === '') {
-    }
-
-    return <div className="markdown-html" dangerouslySetInnerHTML={{ __html: help }} />;
-  }
+  return <div className="markdown-html" dangerouslySetInnerHTML={{ __html: renderedMarkdown }} />;
```
Comment (wbrowne) [inline public/app/core/components/PluginHelp/PluginHelp.tsx:27]: Shall we check if renderedMarkdown is `""`, so we can have a nice message @jackw?

Comment (jackw) [reply public/app/core/components/PluginHelp/PluginHelp.tsx:31]: Yes, well spotted. Have added something in 58bed2e52a69a3cfc5f15920367c595d5ad6cf89

**Comment by wbrowne** [comment]
Tested and LGTM 🚀

**Pull request merged**